app/tests/factories.py

```diff
@@ -54,12 +79,15 @@ class Meta:
     FileId = factory.LazyFunction(uuid.uuid4)
     consignment = factory.SubFactory(ConsignmentFactory)
     FileReference = FuzzyText(length=10)
-    FileType = FuzzyText(length=10)
+    FileType = FuzzyChoice(["File"])
```

Comment:
We only have FileType=file for now
 This avoids generating random strings and removes the need to override FileType in every test.